mkdocs.yml

```diff
@@ -20,7 +20,8 @@ nav:
     - Noise models: examples/startup_guide/4_noise_models.ipynb
     - Objective metrics: examples/startup_guide/5_metrics.ipynb
     - Solvers & algorithms: examples/startup_guide/6_solvers.ipynb
-    - Benchmarking solvers: examples/startup_guide/7_benchmarking.ipynb
+#    - Benchmarking solvers: examples/startup_guide/7_benchmarking.ipynb
+    - Benchmarking: examples/benchmark.ipynb
```

Comment:
I believe the previous one is for benchmarking circuits and utility tools (We had an inaccurate description previously) and the new one is for runtime of TimeReversedSolver as well as simulation time of two backends. Shall we keep both and change "benchmarking" to a more descriptive heading for each?

Reply:
Okay, I kept the old notebook as well and changed the name. Good to merge now.